### ISSUE TITLE
PDF display is too large

### ISSUE THREAD

**Issue by alangrafu** [jellyfin/jellyfin-web#6288]
**Describe The Bug**

Adding PDFs to a books library is great, but when opening the book, it is zoomed and only a small portion of the page is visible. Even zooming out on the browser won't allow me to see the whole page.

**Steps To Reproduce**

1. Go to a PDF book and open it
2. See PDF zoomed in

**Expected Behavior**

The PDF should be presented in a full page fashion.

**Logs**

I don't think there are lot of logs available

```
[18:52:11] [INF] [17] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Saving playback tracking activity in DB
[18:52:11] [INF] [17] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : Exited
[18:52:12] [INF] [17] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:52:33] [INF] [60] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:52:53] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:53:13] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:53:33] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:53:54] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:54:15] [INF] [17] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:54:36] [INF] [60] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
[18:54:56] [INF] [60] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : 73832228-036F-49BA-979A-69B18958D978-eec4f2ec4deb429bb766c14e1045b1a3-3a6b0cd7171b065fbb06ca4539ad2b4
1
```

**Screenshots**

![Image](https://github.com/user-attachments/assets/e0bf04e7-60a3-45cc-9a1e-897555f09386)
**System (please complete the following information):**
 - Platform: Docker container on Debian Bookworm
 - Browser: Librewolf 131.0.3-1
 - Jellyfin Version: 10.10.0

**Additional Context**

**Comment by thornbill** [comment]
I suspect this could be a regression introduced by #6182.

**Comment by dulli** [comment]
> I suspect this could be a regression introduced by [#6182](https://github.com/jellyfin/jellyfin-web/pull/6182).

Yes, seems like I left out some important parts of my original PR when splitting it up into the smaller one, sorry about not catching that!

**Comment by thornbill** [comment]
No worries @dulli it happens. 🙃 

This should be fixed when 10.10.2 is released.

**Comment by alangrafu** [comment]
Thank you!
